repo/db/contacts.go

```diff
@@ -119,6 +152,31 @@ func (c *ContactDB) Count() int {
 	return count
 }
 
+func (c *ContactDB) UpdateUsername(id string, username string) error {
```

Comment:
So does this mean that in theory, a peer could update their local representation of a contact's username or avatar? This is potentially kind of nice.

Reply:
Currently, no, because it's not exposed to any user API, but we could